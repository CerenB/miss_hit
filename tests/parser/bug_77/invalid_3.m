% (c) Copyright 2020 Florian Schanda

parfor ((i = 1:5))
  disp(i);
end
