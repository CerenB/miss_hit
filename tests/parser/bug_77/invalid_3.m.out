In invalid_3.m, line 3
| parfor ((i = 1:5))
|         ^ error: expected IDENTIFIER, found BRA instead
MISS_HIT Summary: 1 file(s) analysed, 1 error(s)
